package.json

```diff
@@ -11,29 +11,23 @@
   "collection:main": "dist/collection/index.js",
   "unpkg": "dist/esn-frontend-application-grid/esn-frontend-application-grid.js",
   "files": [
+    "src/",
     "dist/",
-    "loader/"
+    "loader/",
+    "stencil.config.ts",
+    "tsconfig.json"
   ],
   "scripts": {
     "build": "stencil build --docs",
     "build:watch": "stencil build --watch --serve",
     "start": "stencil build --dev --watch --serve",
-    "test": "stencil test --spec --e2e",
-    "test.watch": "stencil test --spec --e2e --watchAll",
     "generate": "stencil generate",
     "postinstall": "stencil build"
   },
-  "devDependencies": {
-    "@stencil/core": "1.12.2",
-    "@stencil/sass": "1.3.2",
-    "@types/jest": "24.9.1",
-    "@types/puppeteer": "2.0.1",
-    "jest": "24.9.0",
-    "jest-cli": "24.9.0",
-    "puppeteer": "2.1.1"
-  },
   "license": "AGPLv3",
   "dependencies": {
+    "@stencil/core": "1.12.2",
+    "@stencil/sass": "1.3.2",
```

Comment:
These need to be moved from `devDependencies` to `dependencies` so that they also get installed when we do `npm install` in other modules/packages.